<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="VDC1ODCD200101" failures="0" tests="6" name="interview_Questions.timeouts" time="3.191" errors="1" timestamp="2023-09-18T00:22:37 IST" skipped="0">
  <testcase classname="interview_Questions.timeouts" name="test" time="0.005"/>
  <system-out/>
  <testcase classname="interview_Questions.timeouts" name="test" time="0.040"/>
  <system-out/>
  <testcase classname="interview_Questions.timeouts" name="test1" time="3.127">
    <error message="Method interview_Questions.timeouts.test1() didn&#039;t finish within the time-out 3000" type="org.testng.internal.thread.ThreadTimeoutException">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method interview_Questions.timeouts.test1() didn't finish within the time-out 3000
at java.base@17.0.3/java.lang.Thread.sleep(Native Method)
at app//interview_Questions.timeouts.test1(timeouts.java:16)
at java.base@17.0.3/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base@17.0.3/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base@17.0.3/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base@17.0.3/java.lang.reflect.Method.invoke(Method.java:568)
at app//org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at app//org.testng.internal.invokers.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at app//org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:76)
at app//org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:11)
at java.base@17.0.3/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base@17.0.3/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base@17.0.3/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base@17.0.3/java.lang.Thread.run(Thread.java:833)
]]>
    </error>
  </testcase> <!-- test1 -->
  <system-out/>
  <testcase classname="interview_Questions.timeouts" name="test" time="0.004"/>
  <system-out/>
  <testcase classname="interview_Questions.timeouts" name="test" time="0.010"/>
  <system-out/>
  <testcase classname="interview_Questions.timeouts" name="test" time="0.005"/>
  <system-out/>
</testsuite> <!-- interview_Questions.timeouts -->
